packages/hash/frontend/src/components/hooks/blockProtocolFunctions/useBlockProtocolUpdateEntity.ts

```diff
@@ -0,0 +1,73 @@
+import { useMutation } from "@apollo/client";
+import { EmbedderGraphMessageCallbacks } from "@blockprotocol/graph";
+import { updateEntity as updateEntityQuery } from "@hashintel/hash-shared/queries/entity.queries";
+import { updatePage } from "@hashintel/hash-shared/queries/page.queries";
+import { useCallback } from "react";
+
+import {
+  UpdateEntityMutation,
+  UpdateEntityMutationVariables,
+  UpdatePageMutation,
+  UpdatePageMutationVariables,
+} from "../../../graphql/apiTypes.gen";
+import { convertApiEntityToBpEntity } from "../../../lib/entities";
+import { addIdentifiersToMessageData } from "./shared";
+
+export const useBlockProtocolUpdateEntity = (
+  updateForPage: boolean = false,
```

Comment:
Perhaps it's worth adding documentation for this parameter to help explain why it's needed (unless we are planning on getting rid of it soon)

Reply:
Thanks, done in https://github.com/hashintel/hash/pull/752/commits/60e0ea0fb5ccc616e1a5a588bf8cf4e404afb846